engine/cmd/executor/main.go

```diff
@@ -16,15 +16,18 @@ package main
 import (
 	"context"
 	"flag"
+	"fmt"
 	"os"
 	"os/signal"
 	"strings"
 	"syscall"
 
 	"github.com/pingcap/errors"
+	"go.uber.org/zap"
+
```

Comment:
Redundant blank line.

Reply:
The import is grouped by
- standard library
- third party library
- and import path in this repo (this is no standard about this rule, but is widely used in DM code), we need to unify this rule